generated/api/v1/report_configuration_service.swagger.json

```diff
@@ -457,6 +457,12 @@
           "items": {
             "type": "string"
           }
+        },
+        "customSubject": {
```

Comment:
why is the v1 api being changed?

Reply:
The v1 api is the same as storage proto. So, any changes that happen to storage proto also get documented in v1 api. There is a `hidden` option but that is only for the search tags I think.

Reply:
I had the same question